### ISSUE TITLE
Start all sessions nao funciona.

### ISSUE THREAD

**Issue by NoctisBnu** [wppconnect-team/wppconnect-server#1144]
## Description

Estou usando a ultima versão e quando reinicio o server ele nao starta as sessoes q estao aberta, tenho que startar elas pelo swager ou por outro meio. No config.json o start all... esta true.

## Environment

- **Wppconnect-server version(s):** [e.g. 2.0.27, 2.2.1]
- **Browser:** [e.g. Chrome 87, Chromium 85]
- **OS:** [e.g. OSX 10.13.4, Windows 10]
- **Node version:** [e.g. Node 8, Node 14]
- **WhatsApp version:** [e.g. 2.2126.14]
- **MultiDevice (BETA):** yes/no
- **config.json:** [Your src/config.json file content]

## Steps to Reproduce

1. [First Step]
2. [Second Step]
3. [and so on...]

## Log Output

```
If relevant, paste all of your Log Output
```

## Your Code

```
If relevant, paste all of your challenge code in here
```

## Additional context / Screenshot

Add any other context about the problem here. If applicable, add screenshots to help explain.

**Comment by icleitoncosta** [comment]
> ## Description
> Estou usando a ultima versão e quando reinicio o server ele nao starta as sessoes q estao aberta, tenho que startar elas pelo swager ou por outro meio. No config.json o start all... esta true.
> 
> ## Environment
> * **Wppconnect-server version(s):** [e.g. 2.0.27, 2.2.1]
> * **Browser:** [e.g. Chrome 87, Chromium 85]
> * **OS:** [e.g. OSX 10.13.4, Windows 10]
> * **Node version:** [e.g. Node 8, Node 14]
> * **WhatsApp version:** [e.g. 2.2126.14]
> * **MultiDevice (BETA):** yes/no
> * **config.json:** [Your src/config.json file content]
> 
> ## Steps to Reproduce
> 1. [First Step]
> 2. [Second Step]
> 3. [and so on...]
> 
> ## Log Output
> ```
> If relevant, paste all of your Log Output
> ```
> 
> ## Your Code
> ```
> If relevant, paste all of your challenge code in here
> ```
> 
> ## Additional context / Screenshot
> Add any other context about the problem here. If applicable, add screenshots to help explain.

Opa, fiz alguns testes, e o erro aconteceu comigo, vou averiguar aqui para tentar resolver

**Comment by icleitoncosta** [comment]
Fixed by #1145
